crates/proof-of-sql/src/sql/proof_plans/demo_mock_plan.rs

```diff
@@ -0,0 +1,120 @@
+use crate::{
```

Comment:
Is this change meant to be included in this commit?

Reply:
Nope.